### DISCUSSION TITLE
黑色部分如何透明？

### DISCUSSION THREAD

**Discussion by ningdz** [pixijs/pixijs#8562] [Q&A]
如何让黑色部分透明
https://pixijs.io/examples/#/masks/filter.js

**Comment by miltoncandelero** [comment]
https://pixijs.download/dev/docs/PIXI.Application.html

<html>
<body>


options.backgroundAlpha | number | <optional> | 1 | Value from 0 (fully transparent) to 1 (fully opaque).
-- | -- | -- | -- | --



</body>
</html>
